cime_config/config_compsets.xml

```diff
@@ -42,111 +43,95 @@
 
   <compset>
     <alias>F2000climo</alias>
-    <lname>2000_CAM60_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_CISM2%NOEVOLVE_SWAV</lname>
-    <science_support grid="f09_f09_mg17"/>
+    <lname>2000_CAM60_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
   <compset>
     <alias>FHIST</alias>
-    <lname>HIST_CAM60_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_CISM2%NOEVOLVE_SWAV</lname>
-    <science_support grid="f09_f09_mg17"/>
+    <lname>HIST_CAM60_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
   <compset>
     <alias>FHIST_BGC</alias>
-    <lname>HIST_CAM60_CLM50%BGC-CROP_CICE%PRES_DOCN%DOM_MOSART_CISM2%NOEVOLVE_SWAV</lname>
-    <science_support grid="f09_f09_mg17"/>
+    <lname>HIST_CAM60_CLM50%BGC-CROP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
-
   <!-- NorESM compset -->
 
   <!-- comment on the naming of NorESM compsets
 
-	(1) CAM60%NORESM versus CAM60%PTAERO :
-		CAM60%NORESM : the compsets used and created for CMIP6 use "NORESM" to activate typically NorESM2 settings
-		CAM60%PTAERO : older compsets use "PTAERO" to activate typically NorESM2 settings
+  (1) CAM60%NORESM : the compsets used and created for CMIP6 use "NORESM" to activate typically NorESM2 settings
 
-	(2) presence of frc2 (FRC2) in the compset name : 
+	(2) presence of frc2 (FRC2) in the compset name :
```

Comment:
Not absolutely required but here and below (through line 97), can you untabbify?

Reply:
Sorry - I can untabify if you want - but it makes reading the xml file a lot easier.
Actually there are fewew entries now in config_compsets.xml as apposed since the entries related to SSTICE have been removed.